python_modules/dagster-graphql/dagster_graphql/schema/inputs.py

```diff
@@ -365,12 +365,12 @@ class Meta:
     name = graphene.NonNull(graphene.String)
 
 
-class GrapheneInputTag(graphene.InputObjectType):
-    name = graphene.NonNull(graphene.String)
+class GrapheneTagInput(graphene.InputObjectType):
```

Comment:
you should run `yarn generate-graphql` to see if this changes any frontend assumptions